src/dbnode/storage/bootstrap.go

```diff
@@ -117,8 +119,9 @@ func (m *bootstrapManager) IsBootstrapped() bool {
 	return state == Bootstrapped
 }
 
-func (m *bootstrapManager) LastBootstrapCompletionTime() (time.Time, bool) {
-	return m.lastBootstrapCompletionTime, !m.lastBootstrapCompletionTime.IsZero()
+func (m *bootstrapManager) LastBootstrapCompletionTime() (xtime.UnixNano, bool) {
+	bsTime := xtime.UnixNano(m.lastBootstrapCompletionTime.Load())
```

Comment:
Probably could just use the existing sync.RWMutex to protect this state yeah?

Here it seems fine, but just in general for not highly contended code paths would prefer to stick to simple solutions, otherwise folks start to do lots of tricky hand offs and CompareAndSwaps... etc when they could have just as easily used a mutex.

Thoughts?

Reply:
> Probably could just use the existing sync.RWMutex to protect this state yeah?
+1, actually this is how I was fixing it myself.

Reply:
Sure thing! I just assumed it was as gnarly as `flushManager`, so used the same apporach.